Backend/app/dashboard/CommonAPIs/src/Dashboard/ProviderPlatform/Ride.hs

```diff
@@ -134,7 +136,21 @@ newtype EndRideReq = EndRideReq
 instance HideSecrets EndRideReq where
   hideSecrets = identity
 
----------------------------------------------------------
+--------------------------- MultipleRideEndAPI -----------------------------
+type MultipleRideEndAPI =
+  "end"
+    :> ReqBody '[JSON] MultipleRideEndReq
+    :> Post '[JSON] [APISuccess]
+
+newtype MultipleRideEndReq = MultipleRideEndReq
+  { rideLatLong :: [(Id Ride, Maybe LatLong)]
```

Comment:
why its tuple?
maybe it would be better to have explicit fields, like

```
newtype MultipleRideEndReq = MultipleRideEndReq
  { rides :: [MultipleRideItem]
  }

data EndRideReq = EndRideReq
  { rideId :: Id Ride, 
    point :: Maybe LatLong
  }
```